clinica/pipelines/pet/linear/pipeline.py

```diff
@@ -101,6 +106,20 @@ def _build_input_node(self):
             )
 
         # Inputs from t1-linear pipeline
+        # T1w images registered
+        if self.parameters["uncropped_image"]:
+            t1w_linear_file_pattern = T1W_LINEAR
+        else:
+            t1w_linear_file_pattern = T1W_LINEAR_CROPPED
```

Comment:
When calling the pipeline from the CLI, there is indeed an "uncropped_image" entry in the `parameters` dictionary. But when calling from the "Python API" (like in the functional tests for example), there is no default value for it which results in a `KeyError` at this line.
We probably should improve this, but for the sake of this PR, we can provide a default value:

```suggestion
        t1w_linear_file_pattern = T1W_LINEAR if self.parameters.get("uncropped_image", False) else T1W_LINEAR_CROPPED
```